### PR TITLE
Admin design

### PR DESCRIPTION
**Pull request by TaimaTask**

### CODE REVIEW

File: src/components/common/InputField.jsx [hunk 1]
```diff
@@ -30,7 +32,7 @@ const InputField = (props) => {
 
   return (
     <div>
-      <label className={classes.label}>{label}</label>
+      {/* <label className={classes.label}>{label}</label> */}
```
Comment (IZablah) [inline src/components/common/InputField.jsx]: Why Comments? 
**Warning**
 Line 8:5:   'label' is assigned a value but never used  no-unused-vars

File: src/components/common/InputField.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import classNames from "classnames";
+import { Icon } from "react-icons/md";
```
Comment (IZablah) [inline src/components/common/InputField.jsx]: Icon never used? 
**Warning**
Line 3:10:  'Icon' is defined but never used            no-unused-vars

File: src/components/common/InputField.jsx [hunk 3]
```diff
@@ -17,10 +18,11 @@ const InputField = (props) => {
     checkConfirm,
     isButtonClicked,
     autoComplete,
+    Icon,
```
Comment (IZablah) [inline src/components/common/InputField.jsx]: **Warning**

Line 21:5:  'Icon' is assigned a value but never used   no-unused-vars

File: src/components/common/InputField.module.css [hunk 4]
```diff
@@ -11,27 +11,39 @@ body {
...
   background-color: rgba(333, 0, 0, 0.5);
 }
 
 .errorMessage {
   color: #ff0000;
-}
\ No newline at end of file
+}
+
+
```
Comment (IZablah) [inline src/components/common/InputField.module.css]: Remove line

File: src/components/common/backgroundLogo/backgroundLogo.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+const BackgroundLogo = ({title}) => {
+  return (
+    <>
+      <div className={classes.banner}>
+        <img className={classes.schoolLogo} src={schoolLogo} />
+      </div>
+      <div className={classes.sitename}>{title}</div>
+    </>
+  );
+};
```
Comment (IZablah) [inline src/components/common/backgroundLogo/backgroundLogo.jsx:18]: Should add line before export

File: src/components/common/footer/footer.jsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import classes from "./footer.module.css";
+
+const Footer = () => {
+  return (
+<div className={classes.loginFooter}></div>
+  );
+};
```
Comment (IZablah) [inline src/components/common/footer/footer.jsx:8]: Should add line before export

File: src/components/common/footer/footer.jsx [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import classes from "./footer.module.css";
+
+const Footer = () => {
+  return (
+<div className={classes.loginFooter}></div>
```
Comment (IZablah) [inline src/components/common/footer/footer.jsx:6]: Best practices file

File: src/components/mainNavbar.jsx [hunk 8]
```diff
@@ -40,94 +44,114 @@ const MainNavbar = () => {
       .then((querySnapshot) => {
         querySnapshot.forEach((doc) => {
           dispatch(authActions.setUserRole(doc.data()));
-        })
+        });
       })
       .catch((erorr) => {
-        console.log(erorr)
+        console.log(erorr);
```
Comment (IZablah) [inline src/components/mainNavbar.jsx]: Should remove it `console.log`, or replace another feature

File: src/components/mainNavbar.module.css [hunk 9]
```diff
@@ -1,36 +1,95 @@
...
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  background: #f5f5f5;
+  min-height: 200vh;
+}
```
Comment (IZablah) [inline src/components/mainNavbar.module.css]: Should add line here

File: src/components/mainNavbar.module.css [hunk 10]
```diff
@@ -91,3 +161,17 @@
...
+  position: relative;
+  top: 0;
+  left: 0;
+  width: 100%;
+  left: 20%;
+}
+.schoolLogo {
+  height: 80px;
+  top: 30%;
+  /* position: relative; */
```
Comment (IZablah) [inline src/components/mainNavbar.module.css]: Remove comments

File: src/pages/administration/userRole/users.jsx [hunk 11]
```diff
@@ -39,50 +41,66 @@ const Users = () => {
...
             <option value="Administration">Administration</option>
           </select>
-        )
+        ),
       },
+      // {
+      //   Header: "Role",
+      //   accessor: "role",
+      //   Cell: ({ cell: { value } }) => value || "-",
+      // },
```
Comment (IZablah) [inline src/pages/administration/userRole/users.jsx]: Remove comments

File: src/pages/administration/userRole/users.jsx [hunk 12]
```diff
@@ -39,50 +41,66 @@ const Users = () => {
...
     ],
     []
-  )
+  );
+
+  const saveButtonHanler = () => {};
```
Comment (IZablah) [inline src/pages/administration/userRole/users.jsx]: What is that? why is empty

Comment (TaimaTask) [reply src/pages/administration/userRole/users.jsx]: I need it next time when the administrator needs to save changes

**Pull request merged**